webapp/package.json

```diff
@@ -22,6 +22,7 @@
     "@reduxjs/toolkit": "^1.8.5",
     "@tanstack/react-table": "^8.7.0",
     "axios": "^1.1.3",
+    "chakra-react-select": "^4.4.2",
```

Comment:
I'm not sure whether it is my local problem.
When I tried to start the app, it showed 
<img width="764" alt="image" src="https://user-images.githubusercontent.com/1866157/206607334-e7d08fb0-2c14-44bd-926e-72f9a0d5a761.png">
Clean install does not seems to work. 🤔

Reply:
After I switched `npm` to `yarn`, the problem was fixed.

Reply:
These steps work for me 🤔 
```
1) git clone https://github.com/rainy789/centraldogma.git
2) cd centraldogma
3) git fetch origin
4) git checkout --track origin/new-webapp
5) cd webapp
6) npm i
7) npm run mock
```